Content.Server/GameObjects/Components/Sound/SoundEmitterComponent.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Log;
+using Robust.Shared.Utility;
+using YamlDotNet.RepresentationModel;
+using Content.Server.Interfaces;
+using Content.Shared.GameObjects;
+using Robust.Shared.Serialization;
+using Robust.Shared.ViewVariables;
+using Content.Server.GameObjects.EntitySystems;
+using Robust.Server.GameObjects.EntitySystems;
+using Content.Shared.Audio;
+using Robust.Shared.Prototypes;
+using Robust.Shared.IoC;
+using Robust.Shared.Audio;
+
+namespace Content.Server.GameObjects.Components.Sound
+{
+    /// <summary>
+    /// Simple sound emitter that emits sound on use in hand
+    /// </summary>
+    public class SoundEmitterComponent : Component, IUse
```

Comment:
This name sounds too generic. I would go with something more descriptive like `EmitSoundOnUseComponent`.